### PR TITLE
Fixes #354: Overflow menu for multitasking in current session

### LINKED ISSUE #354: FNX3-17093 ⁃ Overflow menu for multitasking in current session
As a user, I'd like to be able to press the overflow button for the current session and get the following options. UX is provided in #239.

* [ ] Delete
* [ ] Archive
* [ ] Send and Share

### PR DESCRIPTION
**Pull request by colintheshots**

### CODE REVIEW

File: app/src/main/res/drawable/ic_archive.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (boek) [inline app/src/main/res/drawable/ic_archive.xml]: nit: license

File: app/src/main/res/drawable/ic_delete.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (boek) [inline app/src/main/res/drawable/ic_delete.xml]: nit: license

File: app/src/main/res/layout/fragment_home.xml [hunk 3]
```diff
@@ -2,79 +2,83 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<androidx.constraintlayout.motion.widget.MotionLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.coordinatorlayout.widget.CoordinatorLayout
```
Comment (boek) [inline app/src/main/res/layout/fragment_home.xml]: Did you mean to wrap this inside a coordinator layout?

Comment (colintheshots) [reply app/src/main/res/layout/fragment_home.xml]: Yes, the bottom sheet behavior depends upon it.

**Comment by colintheshots** [comment]
Do we put licenses inside every layout and Vector Drawable SVG? I thought it was just for code. Someone please correct me if I'm wrong.

File: app/src/main/res/layout/fragment_home.xml (hunk 3 above)
Comment (boek) [reply app/src/main/res/layout/fragment_home.xml]: That’s what I thought as well, but I tested it without it and it seemed to work?

**Comment by boek** [comment]
@colintheshots I'm pretty sure (based on the Focus codebase) we put the license in every xml file.

File: app/src/main/res/layout/fragment_home.xml (hunk 3 above)
Comment (boek) [reply app/src/main/res/layout/fragment_home.xml]: Ah I see the behavior now. I will play around with it.

**Pull request merged**